### PR TITLE
feat: new icon - Globe: network add

### PR DESCRIPTION
**Pull request by williamshammond**
#### Checklist

- [ ] Includes tests (N/A)
- [ ] Update documentation (N/A)

#### Changes proposed in this pull request:
- Adds new icon "Globe: network add" similar to "Globe: network" but with an addition symbol in the top right in the style of many other blueprint components
- Titled "Globe: network add" which is tricky due to the double modifier but follows the style of "Folder: shared open" "Sort: numerical descending" and other icons with two modifier words

#### Reviewers should focus on:
- The title and if the convention makes sense

#### Screenshot
Screenshot 2024-10-03 at 3 12 17 PM

### CODE REVIEW

File: resources/icons/16px/globe-network-add.svg [hunk 1]
```diff
@@ -0,0 +1,4 @@
...
+<path d="M9 5C8.4 5 8 4.6 8 4C8 3.5 8.4 3 9 3H11V1C11 0.4 11.4 0 12 0C12.6 0 13 0.4 13 1V3H15C15.6 3 16 3.4 16 4C16 4.6 15.6 5 15 5H13V7C13 7.6 12.6 8 12 8C11.4 8 11 7.6 11 7V5H9Z" fill="#111418"/>
+</svg>
```
Comment (ggdouglas) [inline resources/icons/16px/globe-network-add.svg:4]: Can we clean this up a bit? We can safely remove the width/height dimensions, fill/clip-rules, and fills:
```svg
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16">
    <path d="M6.693 5a2.4 2.4 0 0 0 .767 1H6.12C6.05 6.61 6 7.27 6 8s.05 1.39.12 2h3.76c.053-.46.094-.947.111-1.472a2.4 2.4 0 0 0 .964.76q-.026.365-.065.712h2.76c.159-.454.27-.93.32-1.422A2.53 2.53 0 0 0 14.5 7v-.5h.5c.28 0 .563-.042.834-.131q.165.79.166 1.631c0 4.42-3.58 8-8 8s-8-3.58-8-8A7.998 7.998 0 0 1 9.631.166 2.7 2.7 0 0 0 9.5 1v.5H9C7.476 1.5 6.5 2.77 6.5 4v.048q-.13.435-.23.952zM5.27 5c.2-1.11.51-2.03.91-2.69A6.03 6.03 0 0 0 2.83 5zM2.35 6C2.13 6.63 2 7.3 2 8s.13 1.37.35 2h2.76C5.04 9.38 5 8.72 5 8s.04-1.38.11-2zm2.92 5H2.83a6.03 6.03 0 0 0 3.35 2.69c-.4-.66-.7-1.58-.91-2.69m1 0c.37 1.9 1.06 3 1.73 3s1.36-1.1 1.73-3zm4.46 0c-.21 1.11-.51 2.03-.91 2.69A6.03 6.03 0 0 0 13.17 11z"/>
    <path d="M9 5c-.6 0-1-.4-1-1 0-.5.4-1 1-1h2V1c0-.6.4-1 1-1s1 .4 1 1v2h2c.6 0 1 .4 1 1s-.4 1-1 1h-2v2c0 .6-.4 1-1 1s-1-.4-1-1V5z"/>
</svg>
```

File: resources/icons/20px/globe-network-add.svg [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+<path d="M13 5C12.4 5 12 4.6 12 4C12 3.5 12.4 3 13 3H15V1C15 0.4 15.4 0 16 0C16.5 0 17 0.4 17 1V3H19C19.5 3 20 3.4 20 4C20 4.6 19.5 5 19 5H17V7C17 7.6 16.5 8 16 8C15.4 8 15 7.6 15 7V5H13Z" fill="#111418"/>
+</svg>
```
Comment (ggdouglas) [inline resources/icons/20px/globe-network-add.svg:4]: Same for this one:
```svg
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20">
  <path d="m12.75 7-.002-.01Q12.874 7 13 7c0 .755.26 1.539.86 2.14q.062.06.125.116A18 18 0 0 1 13.89 12h3.83c.17-.64.28-1.31.28-2q0-.37-.04-.732A2.92 2.92 0 0 0 19 7q.264 0 .526-.049C19.834 7.912 20 8.937 20 10c0 5.52-4.48 10-10 10S0 15.52 0 10 4.48 0 10 0c1.06 0 2.084.165 3.043.472Q13 .737 13 1c-1.049 0-1.88.5-2.398 1.204A1.1 1.1 0 0 0 10 2C8.93 2 7.75 4.05 7.25 7zm.14 5c.07-.64.11-1.31.11-2s-.04-1.36-.12-2H7.11a18.4 18.4 0 0 0 0 4zM6.24 7c.31-1.99.92-3.66 1.72-4.73A8 8 0 0 0 2.61 7zM2 10c0-.69.11-1.36.28-2h3.83a18.4 18.4 0 0 0 0 4H2.28c-.17-.64-.28-1.31-.28-2m4.24 3H2.61c.94 2.31 2.9 4.08 5.35 4.73-.8-1.07-1.41-2.74-1.72-4.73M10 18c-1.07 0-2.25-2.05-2.75-5h5.5c-.5 2.95-1.68 5-2.75 5m3.76-5c-.32 1.99-.93 3.66-1.72 4.73A8 8 0 0 0 17.39 13z"/>
  <path d="M13 5c-.6 0-1-.4-1-1 0-.5.4-1 1-1h2V1c0-.6.4-1 1-1 .5 0 1 .4 1 1v2h2c.5 0 1 .4 1 1s-.5 1-1 1h-2v2c0 .6-.5 1-1 1-.6 0-1-.4-1-1V5z"/>
</svg>

```

File: resources/icons/16px/globe-network-add.svg (hunk 1 above)
Comment (williamshammond) [reply resources/icons/16px/globe-network-add.svg:4]: @ggdouglas Fixed! Thanks

File: resources/icons/20px/globe-network-add.svg (hunk 2 above)
Comment (williamshammond) [reply resources/icons/20px/globe-network-add.svg:4]: @ggdouglas Fixed! I had to keep `fill-rule="evenodd"` or I would get strange patterning in the icon
Screenshot 2024-10-07 at 4 08 19 PM

Comment (ggdouglas) [reply resources/icons/20px/globe-network-add.svg:4]: Ah, my bad! I neglected to visually inspect the path after optimizing the above. Modified the path in this commit to have it comply with the `nonzero` (default) rule: 401a01d3117c744ccfdae005f76ecd0d36e9c2f7

Thanks for getting the optimized versions in!

**Review by ggdouglas — APPROVED** [review]

**Pull request merged**